webapp/i18n/en.json

```
@@ -116,6 +116,7 @@
   "Sidebar.about": "About Focalboard",
   "Sidebar.add-board": "+ Add board",
   "Sidebar.add-template": "New template",
+  "Sidebar.buildInfo": "Commit {commit} Date: {date}",
```

This doesn't match exactly the `defaultMessage` text, so it can lead to confusions.
```suggestion
  "Sidebar.buildInfo": "Commit {commit}\nDate: {date}",
```